css/main-styling.css

```diff
@@ -0,0 +1,89 @@
+body {
+  font-family: 'Roboto Slab', serif;
+  font-size: 18px;
+  line-height: 1.5;
+  cursor: default;
+  font-color: black;
+}
+
+.wrapper {
+  max-width: 1280px;
+  margin: auto;
+}
+
+h1 {
+  font-size: 2em;
+  line-height: 1;
+}
+
+h3 {
+  font-size: 1.25em;
+  font-weight: bold;
+}
+
+p {
+  margin: 0 0 1.5em 0;
+  font-size: 14px;
+}
+
+.button {
+  text-decoration: none;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-weight: bold;
+  color: inherit;
+  display: block;
+  text-align: center;
+  box-shadow: inset 0 -3px 0 rgba(0,0,0,.2);
+  border-radius: 5px;
+}
+
+#page-header {
+  background: #97c0e6;
+  overflow: hidden;
+  padding: 0 3%;
```

Comment:
Generally we want padding to be in px, since it's in comparison to the font size, not the width of the element.